style.css

```diff
@@ -13,7 +13,7 @@ body {
   box-sizing: border-box;
 }
 
-form {
+#form {
   margin-top: 2rem;
   display: flex;
   flex-direction: column;
```

Comment:
- [ ] As I mentioned above you should follow the wireframe properly and shouldn't use the bootstrap too. Please adjust your style by following the design provided by *Microverse*. It seems like your `Add` button is also not aligned properly as per the given design and you used `Bootstrap` for adding the hover effect. Kindly just use CSS for adding the styles or effects. Make sure that your design is matching with given wireframe as much as possible.

### Design Comparison
<table>
<tr>
<th>Given Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-08-12 21-21-58](https://github.com/blessing-michael/books-claases/assets/66476329/153cb74a-1eff-43ff-9034-d770418830c0)

</td>
<td>

![Screenshot from 2023-08-12 21-21-11](https://github.com/blessing-michael/books-claases/assets/66476329/21674005-825d-418c-9a96-25adbc5c0905)

</td>
</tr>
</table>

Reply:
Fixed.